index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="home-container">
+        <div class="mobile-info">
+                <div class="time"><span id="time">9:41</span></div>
+                <span class="mobile-status">
+                    <img src="../Personal-portfolio/icons/connection.png" alt="network">
+                    <img src="../Personal-portfolio/icons/wifi.png" alt="wifi">
+                    <img src="../Personal-portfolio/icons/battery.png" alt="battery">
+                </span>
+            </div>
+    <div class="main-content">
+        <div class="logo">
+            <span>My logo</span>
+            <a>
+                <img src="../Personal-portfolio/icons/menu.png" alt="menu">
+            </a>
+        </div>
+
+        <div class="main-info">
```

Comment:
- [ ] Please the container of this section must be a `<section>` tag instead of `<div>` because using `<div>` does not define a section in HTML